### PR TITLE
165132527-track-gender-info-in-forecast-api

### PR DESCRIPTION
**Pull request by c-troha**
Male/Female count properties added to snapshots, seeder, tests, and poller to successfully track that data.

### CODE REVIEW

File: src/Housing.Forecast.Service/Controllers/ForecastController.cs [hunk 1]
```diff
@@ -408,15 +410,17 @@ private async Task<List<Snapshot>> CreateSnapshots(DateTime? date, string locati
...
             var snapshot = new Snapshot()
             {
                 Date = date,
                 RoomOccupancyCount = rooms,
                 UserCount = users,
                 Location = location,
-                Created = DateTime.Now
+                Created = DateTime.Now,
+                MaleCount = mCount,
+                FemaleCount = fCount
```
Comment (escalonn) [inline src/Housing.Forecast.Service/Controllers/ForecastController.cs]: this method looks a bit silly at first glance, consider removing it on some cleanup user story

File: src/Housing.Forecast.Service/Controllers/ForecastController.cs [hunk 2]
```diff
@@ -353,7 +353,8 @@ private async Task<List<Snapshot>> CreateSnapshots(DateTime? date, string locati
                     rooms = await _room.GetByDateAsync(date.Value);
                     users = await _user.GetByDateAsync(date.Value);
 
-                    Snapshot snapshot = CreateSnapshot(date.Value, rooms.Select(x => x.Occupancy).Sum(), users.Count, string.IsNullOrEmpty(location) ? "All" : text.ToTitleCase(location));
+                    Snapshot snapshot = CreateSnapshot(date.Value, rooms.Select(x => x.Occupancy).Sum(), users.Count, string.IsNullOrEmpty(location) ? "All" : text.ToTitleCase(location),
+                        users.Count(x => x.Gender.Equals("M")), users.Count(x => x.Gender.Equals("F")));
```
Comment (escalonn) [inline src/Housing.Forecast.Service/Controllers/ForecastController.cs]: this is pretty painful. while we are editing it let's put some line breaks in here, move the ternary statement to a local variable. same for the similar line(s) elsewhere in file.

**Comment by escalonn** [comment]
also merge from broker

File: src/Housing.Forecast.Service/Controllers/ForecastController.cs (hunk 2 above)
Comment (stormrubio3000) [reply src/Housing.Forecast.Service/Controllers/ForecastController.cs]: This still needs to be broken up and formatted in an easier to read manner.

**Comment by escalonn** [comment]
merge from broker

**Pull request merged**